var/spack/repos/builtin/packages/python/package.py

```diff
@@ -678,14 +678,20 @@ def libs(self):
 
     @property
     def headers(self):
-        config_h = self.get_config_h_filename()
+        try:
```

Comment:
We have to be careful here, as Python may be installed during cross-compilation, but the Python executable can't be run to query for the header location. Use a try-except to handle this just like we did for `self.home`.